packages/contracts/test/libraries/lib_bytes.ts

```diff
@@ -57,47 +57,47 @@ describe('LibBytes', () => {
 
     describe('areBytesEqual', () => {
         it('should return true if byte arrays are equal (both arrays < 32 bytes)', async () => {
-            const areBytesEqual = await libBytes.areBytesEqual.callAsync(
+            const areBytesEqual = await libBytes.publicAreBytesEqual.callAsync(
```

Comment:
Why the `public` prefix? What does it mean?

Reply:
We're just wrapping all of our utils with public functions so that we can test them in isolation.